src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,144 @@
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+		true,
+		defaults, {},
+		window[ componentName ],
+		wb.getData( $elm, componentName )
+	);
+			i18nDict = i18nDict[ $( "html" ).attr( "lang" ) || "en" ];
+			i18nDict = {
+				warning: i18nDict[ "exitscript-warning" ],
+				exitMsg: i18nDict[ "exitscript-exitMsg" ],
+				yesMsg: i18nDict[ "exitscript-yesMsg" ],
+				cancelMsg: i18nDict[ "exitscript-cancelMsg" ]
+			};
+			$elm.trigger( "setModaldialog", settings );
+			var exitParam = "?" + encodeURIComponent( "targetUrl=" + String( this.href ) );
+
+			if ( settings.targetBlank ) {
+				exitParam = exitParam + encodeURIComponent( "&&targetBlank=true" );
+			}
+
+			// Set default propertie
+
+			if ( settings.displayModal !== false ) {
+				if ( !settings.targetBlank ) {
+					settings.targetBlank = "";
+				} else {
+					settings.targetBlank = "target='_blank'";
+				}
+				if ( !settings.exitMsg ) {
+					settings.exitMsg = i18nDict.exitMsg;
+				}
+				if ( !settings.yesMsg ) {
+					settings.yesMsg = i18nDict.yesMsg;
+				}
+				if ( !settings.cancelMsg ) {
+					settings.cancelMsg = i18nDict.cancelMsg;
+				}
+				var tpl = document.createElement( "div" );
+				tpl.id = "tm";
+				tpl.innerHTML = "<section id='sExitModal' class='mfp-hide modal-dialog modal-content overlay-def'>" +
+			"<header class='modal-header'><h2 class='modal-title'>" + i18nDict.warning + "</h2></header>" +
+			"<div class='modal-body'>" +
+			"<p>" + settings.exitMsg + "</p>" +
+			"<ul class='list-inline text-center'>" +
+			"<li><button class='btn btn-primary popup-modal-dismiss pull-left' id='eCancel'>" + settings.cancelMsg + "</button></li>" +
+			"<li><a class='btn btn-default pull-right popup-modal-close' " + settings.targetBlank + " href='" + settings.exitURL + exitParam + "' type='button'>" + settings.yesMsg + "</a></li>" +
+			"</ul></div></section>";
+				var moDal = document.createDocumentFragment();
+				moDal.appendChild( tpl );
+				document.body.appendChild( moDal );
+
+		//Unwrap div id="tm"
+				var wrapper = document.querySelector( "#tm" );
+				wrapper.outerHTML = wrapper.innerHTML;
+			}
+			wb.ready( $elm, componentName );
+		}
+
+	};
+
+// Add your plugin event handler
+$document.on( "click", selector, function( ev ) {
+	ev.preventDefault();
+
+	var settings = $.extend(
+	defaults, {}
+);
+
+	var exitParam = "?" + encodeURIComponent( "targetUrl=" + String( this.href ) );
+
+	if ( settings.targetBlank ) {
+		exitParam = exitParam + encodeURIComponent( "&&targetBlank=true" );
+	}
+
+	if ( settings.displayModal !== false ) {
+		$document.trigger( "open.wb-lbx", [
+			[ {
+				src: "#sExitModal",
+				type: "inline"
+			} ],
+			true
+		] );
+
+	} else {
+
+		if ( settings.displayModal === false && settings.targetBlank !== true ) {
+			self.location.href = ( settings.exitURL +  exitParam );
+		} else {
+			window.open( settings.exitURL + exitParam );
+		}
+
+	}
+
+} );
+
+$( document ).on( "click", ".popup-modal-close", function() {
```

Comment:
Why, the modal dialog already manage that?

Reply:
what is the status of this

Reply:
This click function need to be removed or move in the right plugin

Reply:
Did you still need this "click" function? even after you added into the modal footer?

Reply:
@duboisp Yes. I need the "click" function. I can move the popup-modal-close to lightbox plugin or edit popup-modal-dismiss.

Reply:
I tried just to overwrite the close button by adding my own modal-footer in the lightbox working example, and I see your issue. However, that fix must be done in lightbox and somehow the lightbox plugin overwrite the modal footer set by the author. 

That fix must be submitted in a different PR because it is not related directly to the exit script plugin but to a bug in lightbox plugin.

Reply:
@duboisp The class is popup-modal-dismiss closes the modal window but the anchor is not working.

Reply:
When the button is changed for an anchor? but you only have an anchor when the person would navigate to the other page. Closing the dialog is useless when the navigation happen in the same tab, no? 

The click event handler support the anchor element.
I saw that it prevent default if it don't contains a target attribute.
ref. https://github.com/wet-boew/wet-boew/blob/master/src/plugins/lightbox/lightbox.js#L348
That behaviour was added by you - https://github.com/wet-boew/wet-boew/commit/5288ca529de6a40e55d061119da5aca449e69f26

The solution will be to define a target "_self" to open the link in the page, but to close the dialog.

The implementation of the dismiss button in the lightbox do the same as the code that was added.

Reply:
@duboisp Pierre.  You asked me to add this behaviour (prevent default if it don't contains a target attribute.) to the lightbox plugin.

Reply:
@duboisp Can we just remove this behaviour (prevent default if it don't contains a target attribute)? It will fix the issue.

Reply:
@ipaksc Do you mean removing the event handler from line 137 to 139? Yes because it doesn't belong in here.